eva/configuration/config_utils.py

```diff
@@ -12,14 +12,28 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from pathlib import Path
 from typing import Any
 
+import yaml
 
-def read_value_config(cfg: Any, category: str, key: str) -> Any:
-    return cfg.get(category, {}).get(key)
 
+def read_value_config(config_path: Path, category: str, key: str) -> Any:
+    with config_path.open("r") as yml_file:
+        config_obj = yaml.load(yml_file, Loader=yaml.FullLoader)
+        if config_obj is None:
+            raise ValueError(f"Invalid path to config file {config_path}")
+        return config_obj[category][key]
 
-def update_value_config(cfg: Any, category: str, key: str, value: str):
-    category_data = cfg.get(category, None)
-    if category_data:
-        category_data[key] = value
+
+def update_value_config(config_path: Path, category: str, key: str, value: str):
```

Comment:
Do we need to make it atomic?

Reply:
Yes, but that will be in the next to next PR